scripts/erlang-setup.ps1

```diff
@@ -12,24 +12,22 @@ $TEMPLATES_DIR = Join-Path $PSScriptRoot "templates"
 
 
 function Install-VCredist {
```

Comment:
This function can be removed since you declared it in the main `DCOSWindowsAgentSetup.ps1` script.

Reply:
will do